src/styles/theme.js

```diff
@@ -9,6 +9,13 @@ let theme = createTheme({
     secondary: {
       main: '#ffc4bd'
     }
+  },
+  overrides: {
+    MuiBottomNavigation: {
+      root: {
+        backgroundColor: '#FFCFCA'
```

Comment:
Maybe I should start looking into exporting these hex values as global variables 🤔